File: dicrete-math/number-theory/parity/go/01/main.go
package main

import "fmt"

func main(){
	var number = 7458755

	if isEven (number) == true {

	fmt.Printf("The number %d is even\n", number)
} else {
	fmt.Printf("The number %d is odd\n", number)
}
}
	//This function returns a boolean value
func isEven(num int) bool{

	if num%2 == 0 {
		return true
	} else {
		return false
	}
}

File: dicrete-math/number-theory/parity/js/01/main.js
var number = 2257;

// console.log(isEven(1023));
if (isEven(number)) {
  console.log(`The number ${number} is even`);
} else {
  console.log(`The number ${number} is odd`);
}
function isEven(num) {
  if (num % 2 == 0) {
    return true;
  } else {
    return false;
  }
}
